packages/shared/src/index.ts

```diff
@@ -45,6 +45,7 @@ export const hasOwn = (
   key: string | symbol
 ): key is keyof typeof val => hasOwnProperty.call(val, key)
 
+export const isDefined = (val: any): boolean => typeof val !== 'undefined'
```

Comment:
Can be made type safe:
```suggestion
export const isDefined = <T extends unknown>(value: T | undefined): value is Exclude<T, undefined> => typeof value !== 'undefined'
```